Ajout de la sauvegarde + niveaux + fix fenêtres

Thotoman opened the pull request:
changement niveaux :
- level2 : rectangles rouges devenus scie
- level3 1 : pareil
- level4 1 : changement des plateforms du jump
- level6 : plus beau / realiste

changement de la couleur du sol pour les niveaux considérés comme dangereux (orange)

creation du level5 1 : comme level 5 mais avec un monstre pic qui tue et un rino qui bouge et le joueur le suit quand il est dessus (par contre quand le rino tourne le joueur avance a l'envers)


/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\
je sais pas pk mais je peux plus lancer le level 1 car il y a un conflit de merge depuis l'ajout de Lucas

Itchybum-kasuga committed:
Bouton Confirmer et fix des fenêtre

dans le titre

ilo80 committed:
Ajout du système de sauvegarde

ilo80 committed:
Merge branch 'dev' of https://github.com/Hack-n-Leap/game into dev

ilo80 committed:
Fix erreur scène

ilo80 committed:
Association de GameManager au joueur

Thotoman committed:
modif level

changement niveaux :
- level2 : rectangles rouges devenus scie
- level3 1 : pareil
- level4 1 : changement des plateforms du jump
- level6 : plus beau / realiste

changement de la couleur du sol pour les niveaux considérés comme dangereux (orange)

creation du level5 1 : comme level 5 mais avec un monstre pic qui tue et un rino qui bouge et le joueur le suit quand il est dessus (par contre quand le rino tourne le joueur avance a l'envers)

ilo80 committed:
Ajout de la récupération de la position

ilo80 committed:
Ajout de toutes les scènes lors de la sauvegarde

ilo80 committed:
Ajout de l'enregistrement des niveaux

ilo80 committed:
Merge branch 'dev' of https://github.com/Hack-n-Leap/game into dev

Thotoman committed:
modif saut

utilise les "pieds" pour detecter le sol et le Layer (a coté de tag) "Ground"

ilo80 committed:
Ajout bouton récupération texte (lvl 1)

ilo80 committed:
Ajout layer ground

ilo80 committed:
Fix jump

ilo80 reviewed (changes requested):
Il faut fix les pb de fenêtres et ajouter la récupération du texte dans tous les niveaux.

ilo80 committed:
Ajout d'ancres pour la taille de la fenêtre (lvl 1)

Thotoman committed:
nv lvl

creation level7 qui recap tout ce qu'a appris le joueur
+ modif gamemanager

ilo80 committed:
Suppression ancres (résolution de conflit)

Thotoman committed:
Merge branch 'dev' of https://github.com/Hack-n-Leap/game into dev

ilo80 committed:
Fix taille fenêtre texte

ilo80 committed:
Merge branch 'dev' of https://github.com/Hack-n-Leap/game into dev

Thotoman committed:
modif cam + level

la cam est moins haute car quand on sautait on voyait plus le sol
donc jai modif chaque level pour pas que ca fasse moche
+ continué le level7
